### ISSUE TITLE
Cant De-Husk someone thats augmented

### ISSUE THREAD

**Issue by bloxrocks** [tgstation/tgstation#61111]
## Round ID:
168685

## Reproduction:
During the round a space-dragon showed up and so there were a lot of husks. i thought i could do a full aug to de-husk but apparently not. i went and got some synthflesh and i couldnt use it since they were augmented. i had to re-attach a flesh limb to de-husk.

**Comment by timothymtorres** [comment]
I'm not sure if this is intended or not.  Maybe this is should be a drawback to having augmented limbs?  Or maybe fully augmented limbs should prevent you from being husked.

**Comment by newfren** [comment]
does rezadone work?

**Comment by Colovorat** [comment]
first of all why you thought augmenting de-husk, second of all augmenting shouldn't prevent usage of synthflesh de-husking of a body. i think it's a bug that you can't use chooms on augmented people

**Comment by ArcaneDefence** [comment]
Makes sense to me that we would be able to fix husking by full augmentation as husking is a damage of the flesh-
but iunno if anyone wants to do that.

**Comment by necromanceranne** [comment]
It's possibly a balance change and less of a fix, but I'm not going to disagree that full augmentation should fix husking.

**Comment by Sealed101** [comment]
augmenting a husk then using rezadone _with a dropper while aiming for the eyes_ dehusks just fine, while synthflesh does not indeed work

**Comment by necromanceranne** [comment]
what method did you use? Patch, splash?

**Comment by Sealed101** [comment]
dropper is basically an injection (it's written in cursive, come on) 
to be fair i was also testing synthflesh with a dropper, which is stupid, but i don't think it would work anyway, if i remember how reagent exposure via splash/spray aka touch works

**Comment by necromanceranne** [comment]
I was asking about the synthflesh, not the rezadone. I think synthflesh only works on touch, patch or vapour.

**Comment by Sealed101** [comment]
just tested and synthflesh foam works of all things. sprays don't work
